src/main/java/org/gatorgradle/task/GatorGradleReportTask.java

```diff
@@ -0,0 +1,123 @@
+package org.gatorgradle.task;
+
+import org.gatorgradle.task.GatorGradleTask;
+import org.gatorgradle.command.BasicCommand;
+import org.gatorgradle.command.Command;
+import org.gatorgradle.config.GatorGradleConfig;
+import org.gatorgradle.display.CommandOutputSummary;
+import org.gatorgradle.internal.DependencyManager;
+import org.gatorgradle.internal.ProgressLoggerWrapper;
+import org.gatorgradle.util.Console;
+
+import org.gradle.api.GradleException;
+import org.gradle.api.tasks.TaskAction;
+import org.gradle.workers.WorkerExecutor;
+
+public class GatorGradleReportTask extends GatorGradleTask {
+
+  @Inject
+  public GatorGradleReportTask(WorkerExecutor executor) {
```

Comment:
A report should not need a task executor -- that is only applicable to the grading task.